### PR TITLE
create the `task` sub-command 

### PR DESCRIPTION
**Pull request by alfredodeza**
This command would allow to query tasks by their identifier, including a `--poll` flag that will keep polling until the task completes.

This PR also adds `requests` as a dependency.

### CODE REVIEW

File: ceph_installer/cli/constants.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+server_address = 'http://localhost:8181/'
```
Comment (ktdreyer) [inline ceph_installer/cli/constants.py]: with #93 this will be `:8080`

File: config/config.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # Server Specific Configurations
 server = {
-    'port': '8080',
+    'port': '8181',
```
Comment (ktdreyer) [inline config/config.py]: conflicts with #93

File: setup.py [hunk 3]
```diff
@@ -64,6 +64,7 @@ def run(self):
         'pecan-notario',
         'notario>=0.0.11',
         'tambo',
+        'requests',
```
Comment (ktdreyer) [inline setup.py]: Mind adding `Requires: python-requests` to `ceph-installer.spec` ?

File: ceph_installer/cli/task.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
+from os import path
+import sys
+import requests
+import time
+from textwrap import dedent
+from tambo import Transport
+
+from ceph_installer import process
```
Comment (ktdreyer) [inline ceph_installer/cli/task.py]: flake8 says this import is unused

File: ceph_installer/cli/task.py [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+this_dir = path.abspath(path.dirname(__file__))
+top_dir = path.dirname(path.dirname(this_dir))
+playbook_path = path.join(top_dir, 'deploy/playbooks')
+
+
+class Task(object):
+
+    help = "/api/tasks/ operations"
+    options = [] #['--poll', 'stdout', 'stderr']
```
Comment (andrewschoen) [inline ceph_installer/cli/task.py]: Do you want these commented out here?

File: ceph_installer/cli/task.py [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+import time
+from textwrap import dedent
+from tambo import Transport
+
+from ceph_installer import process
+from ceph_installer.cli import log, constants
+
+this_dir = path.abspath(path.dirname(__file__))
+top_dir = path.dirname(path.dirname(this_dir))
+playbook_path = path.join(top_dir, 'deploy/playbooks')
```
Comment (andrewschoen) [inline ceph_installer/cli/task.py]: Are you using this anywhere?

File: ceph_installer/cli/task.py (hunk 5 above)
Comment (alfredodeza) [inline ceph_installer/cli/task.py]: ouch, good catch

**Pull request merged**